.circleci/config.yml

```
@@ -487,6 +503,38 @@ jobs:
           replace: $SNAPSHOT_VERSION
       - notify_github:
           data: "$(jq -Rcs '{ body: . }' scripts/snapshot-comment.md)"
+  build_android_release:
+    working_directory: ~/project
+    executor:
+      name: android/android-machine
+      tag: '2024.01.1'
+    steps:
+      - advanced-checkout/shallow-checkout
+      - pack_and_build_sdk
+      - run:
+          name: Build Android
+          command: |
+            cd examples/hybrid/android
+            ./gradlew assembleDebug
```

I believe this job as well doesn't have to be restricted to the hybrid example app. We can also build the app in release mode instead. I also feel like `build_check_android` would be a more suitable name, what do you think?

```suggestion
  build_check_android:
    working_directory: ~/project
    executor:
      name: android/android-machine
      tag: '2024.01.1'
    parameters:
      app_dir:
        type: string
        default: examples/hybrid
    steps:
      - advanced-checkout/shallow-checkout
      - pack_and_build_sdk
      - run:
          name: Build Android
          command: ./gradlew assembleRelease
          working_directory: <<parameters.app_dir>>/android
```